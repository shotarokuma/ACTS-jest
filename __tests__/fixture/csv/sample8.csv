Column1,Column2,Column3,Column4,Column5,Column6,Column7,Column8,Column9,Column10,Column11,Column12,Column13,Column14,Column15,Column16,Column17,Column18,Column19,Column20,Column21,Column22,Column23,Column24,Column25,Column26,Column27,Column28,Column29,Column30
,"sample cell,","sample cell,",,"sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,","sample cell,",,"sample cell,","sample cell,",,"sample cell,"
"sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,,,"sample cell,","sample cell,",,,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,"
"sample cell,",,,"sample cell,","sample cell,",,,"sample cell,","sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,",,,"sample cell,","sample cell,",,,,,"sample cell,","sample cell,","sample cell,",,,"sample cell,"
"sample cell,",,"sample cell,","sample cell,","sample cell,",,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,",,,,,"sample cell,","sample cell,"
"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,","sample cell,",,"sample cell,",,,,"sample cell,","sample cell,",,"sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,,"sample cell,","sample cell,",,,"sample cell,"
"sample cell,",,,,,"sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,",,,,,,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,,"sample cell,",,"sample cell,",,"sample cell,","sample cell,"
,,,,"sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,,,,,,,,,,,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,"
,"sample cell,","sample cell,","sample cell,",,,,,,"sample cell,",,,,,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,",,,,,,"sample cell,",,
,,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,,,,,,,,,,,"sample cell,",,"sample cell,","sample cell,",,,"sample cell,",,"sample cell,"
"sample cell,","sample cell,",,,,,"sample cell,","sample cell,",,,"sample cell,",,,,"sample cell,","sample cell,","sample cell,",,,,,,,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,"
,,"sample cell,",,"sample cell,",,,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,","sample cell,",,"sample cell,",,"sample cell,",
,"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,,"sample cell,","sample cell,",,
,,,"sample cell,","sample cell,",,,,,"sample cell,","sample cell,",,,"sample cell,",,,,,"sample cell,",,,,"sample cell,",,,,"sample cell,",,"sample cell,",
,,,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,"sample cell,",,,"sample cell,",,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,",
"sample cell,",,,,"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,,"sample cell,","sample cell,",,"sample cell,",,,"sample cell,",,"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,"
"sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,"sample cell,",,,,"sample cell,",,"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,",,"sample cell,","sample cell,",,,,"sample cell,",,"sample cell,"
,,,,"sample cell,","sample cell,",,,,"sample cell,",,"sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,,,,"sample cell,",,,"sample cell,",,,"sample cell,",
,"sample cell,","sample cell,","sample cell,","sample cell,",,,,,"sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,","sample cell,",,,,,"sample cell,","sample cell,"
,"sample cell,",,"sample cell,",,,"sample cell,",,,,,"sample cell,",,"sample cell,",,,,,,,"sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,"sample cell,",
"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,,,"sample cell,"
"sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,,,"sample cell,","sample cell,",,"sample cell,","sample cell,",,"sample cell,",,,"sample cell,",,,,"sample cell,","sample cell,",,,,"sample cell,",
,"sample cell,","sample cell,",,,"sample cell,",,"sample cell,",,"sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,,"sample cell,",,,,"sample cell,","sample cell,",,"sample cell,","sample cell,",,,
,"sample cell,","sample cell,","sample cell,",,,,,"sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,",,,,,,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,","sample cell,","sample cell,"
,,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,","sample cell,",,,,"sample cell,",,,"sample cell,",,"sample cell,","sample cell,",,"sample cell,",,"sample cell,"
"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,",,,,,"sample cell,",,"sample cell,","sample cell,",,"sample cell,","sample cell,",,,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,","sample cell,",,"sample cell,","sample cell,"
,"sample cell,",,,,"sample cell,",,,"sample cell,",,,,"sample cell,",,"sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,","sample cell,","sample cell,",,,,"sample cell,",,"sample cell,","sample cell,"
,,"sample cell,",,,,"sample cell,",,"sample cell,",,,,"sample cell,",,"sample cell,",,"sample cell,","sample cell,",,"sample cell,","sample cell,","sample cell,",,,,,"sample cell,",,,
,"sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,",,,"sample cell,","sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,,,"sample cell,",,"sample cell,","sample cell,",,,,"sample cell,","sample cell,","sample cell,"
,,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,"sample cell,",,,,"sample cell,",,"sample cell,",,,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,"
,,,,"sample cell,",,"sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,","sample cell,",,"sample cell,",,"sample cell,",,"sample cell,","sample cell,","sample cell,",,,"sample cell,",,,,,,
