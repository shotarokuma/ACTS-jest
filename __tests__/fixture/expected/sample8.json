[
{"Column1":"","Column2":"sample cell,","Column3":"sample cell,","Column4":"","Column5":"sample cell,","Column6":"sample cell,","Column7":"","Column8":"sample cell,","Column9":"","Column10":"sample cell,","Column11":"","Column12":"","Column13":"sample cell,","Column14":"sample cell,","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"","Column19":"sample cell,","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"sample cell,","Column25":"sample cell,","Column26":"","Column27":"sample cell,","Column28":"sample cell,","Column29":"","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":"","Column6":"sample cell,","Column7":"","Column8":"sample cell,","Column9":"","Column10":"","Column11":"","Column12":"","Column13":"sample cell,","Column14":"sample cell,","Column15":"","Column16":"","Column17":"sample cell,","Column18":"","Column19":"","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"","Column25":"sample cell,","Column26":"sample cell,","Column27":"sample cell,","Column28":"sample cell,","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"","Column3":"","Column4":"sample cell,","Column5":"sample cell,","Column6":"","Column7":"","Column8":"sample cell,","Column9":"sample cell,","Column10":"","Column11":"","Column12":"sample cell,","Column13":"sample cell,","Column14":"sample cell,","Column15":"","Column16":"sample cell,","Column17":"","Column18":"","Column19":"sample cell,","Column20":"sample cell,","Column21":"","Column22":"","Column23":"","Column24":"","Column25":"sample cell,","Column26":"sample cell,","Column27":"sample cell,","Column28":"","Column29":"","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"","Column3":"sample cell,","Column4":"sample cell,","Column5":"sample cell,","Column6":"","Column7":"sample cell,","Column8":"","Column9":"","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"","Column14":"sample cell,","Column15":"sample cell,","Column16":"sample cell,","Column17":"","Column18":"","Column19":"sample cell,","Column20":"sample cell,","Column21":"","Column22":"sample cell,","Column23":"sample cell,","Column24":"sample cell,","Column25":"","Column26":"","Column27":"","Column28":"","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":"sample cell,","Column6":"","Column7":"sample cell,","Column8":"sample cell,","Column9":"","Column10":"sample cell,","Column11":"","Column12":"","Column13":"","Column14":"sample cell,","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"sample cell,","Column19":"","Column20":"sample cell,","Column21":"","Column22":"sample cell,","Column23":"","Column24":"","Column25":"","Column26":"sample cell,","Column27":"sample cell,","Column28":"","Column29":"","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"","Column3":"","Column4":"","Column5":"","Column6":"sample cell,","Column7":"","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"","Column17":"sample cell,","Column18":"","Column19":"","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"","Column25":"sample cell,","Column26":"","Column27":"sample cell,","Column28":"","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"","Column9":"sample cell,","Column10":"","Column11":"sample cell,","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"","Column21":"","Column22":"","Column23":"","Column24":"sample cell,","Column25":"sample cell,","Column26":"sample cell,","Column27":"","Column28":"","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"sample cell,","Column11":"","Column12":"","Column13":"","Column14":"","Column15":"sample cell,","Column16":"","Column17":"","Column18":"sample cell,","Column19":"sample cell,","Column20":"sample cell,","Column21":"","Column22":"sample cell,","Column23":"","Column24":"","Column25":"","Column26":"","Column27":"","Column28":"sample cell,","Column29":"","Column30":""},
{"Column1":"","Column2":"","Column3":"sample cell,","Column4":"sample cell,","Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"sample cell,","Column9":"","Column10":"sample cell,","Column11":"","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"","Column21":"","Column22":"sample cell,","Column23":"","Column24":"sample cell,","Column25":"sample cell,","Column26":"","Column27":"","Column28":"sample cell,","Column29":"","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"sample cell,","Column8":"sample cell,","Column9":"","Column10":"","Column11":"sample cell,","Column12":"","Column13":"","Column14":"","Column15":"sample cell,","Column16":"sample cell,","Column17":"sample cell,","Column18":"","Column19":"","Column20":"","Column21":"","Column22":"","Column23":"","Column24":"sample cell,","Column25":"sample cell,","Column26":"sample cell,","Column27":"","Column28":"","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"","Column3":"sample cell,","Column4":"","Column5":"sample cell,","Column6":"","Column7":"","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"","Column14":"","Column15":"sample cell,","Column16":"sample cell,","Column17":"sample cell,","Column18":"","Column19":"","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"sample cell,","Column25":"sample cell,","Column26":"","Column27":"sample cell,","Column28":"","Column29":"sample cell,","Column30":""},
{"Column1":"","Column2":"sample cell,","Column3":"sample cell,","Column4":"","Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"","Column9":"","Column10":"sample cell,","Column11":"sample cell,","Column12":"","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"","Column19":"sample cell,","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"sample cell,","Column25":"","Column26":"","Column27":"sample cell,","Column28":"sample cell,","Column29":"","Column30":""},
{"Column1":"","Column2":"","Column3":"","Column4":"sample cell,","Column5":"sample cell,","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"sample cell,","Column11":"sample cell,","Column12":"","Column13":"","Column14":"sample cell,","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"sample cell,","Column20":"","Column21":"","Column22":"","Column23":"sample cell,","Column24":"","Column25":"","Column26":"","Column27":"sample cell,","Column28":"","Column29":"sample cell,","Column30":""},
{"Column1":"","Column2":"","Column3":"","Column4":"sample cell,","Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"","Column12":"sample cell,","Column13":"","Column14":"sample cell,","Column15":"","Column16":"","Column17":"sample cell,","Column18":"","Column19":"","Column20":"sample cell,","Column21":"","Column22":"sample cell,","Column23":"","Column24":"","Column25":"sample cell,","Column26":"sample cell,","Column27":"sample cell,","Column28":"","Column29":"sample cell,","Column30":""},
{"Column1":"sample cell,","Column2":"","Column3":"","Column4":"","Column5":"sample cell,","Column6":"sample cell,","Column7":"","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"","Column14":"sample cell,","Column15":"","Column16":"","Column17":"sample cell,","Column18":"sample cell,","Column19":"","Column20":"sample cell,","Column21":"","Column22":"","Column23":"sample cell,","Column24":"","Column25":"sample cell,","Column26":"sample cell,","Column27":"","Column28":"sample cell,","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"sample cell,","Column4":"","Column5":"sample cell,","Column6":"","Column7":"sample cell,","Column8":"","Column9":"","Column10":"sample cell,","Column11":"","Column12":"","Column13":"","Column14":"sample cell,","Column15":"","Column16":"sample cell,","Column17":"sample cell,","Column18":"","Column19":"sample cell,","Column20":"sample cell,","Column21":"sample cell,","Column22":"","Column23":"sample cell,","Column24":"sample cell,","Column25":"","Column26":"","Column27":"","Column28":"sample cell,","Column29":"","Column30":"sample cell,"},
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":"sample cell,","Column6":"sample cell,","Column7":"","Column8":"","Column9":"","Column10":"sample cell,","Column11":"","Column12":"sample cell,","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"","Column19":"","Column20":"","Column21":"","Column22":"","Column23":"sample cell,","Column24":"","Column25":"","Column26":"sample cell,","Column27":"","Column28":"","Column29":"sample cell,","Column30":""},
{"Column1":"","Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":"sample cell,","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"","Column19":"sample cell,","Column20":"","Column21":"sample cell,","Column22":"","Column23":"sample cell,","Column24":"sample cell,","Column25":"","Column26":"","Column27":"","Column28":"","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"sample cell,","Column3":"","Column4":"sample cell,","Column5":"","Column6":"","Column7":"sample cell,","Column8":"","Column9":"","Column10":"","Column11":"","Column12":"sample cell,","Column13":"","Column14":"sample cell,","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"sample cell,","Column25":"","Column26":"sample cell,","Column27":"","Column28":"","Column29":"sample cell,","Column30":""},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"","Column4":"sample cell,","Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"","Column14":"sample cell,","Column15":"","Column16":"sample cell,","Column17":"","Column18":"sample cell,","Column19":"","Column20":"","Column21":"sample cell,","Column22":"","Column23":"","Column24":"sample cell,","Column25":"sample cell,","Column26":"sample cell,","Column27":"","Column28":"","Column29":"","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"","Column3":"sample cell,","Column4":"sample cell,","Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"","Column9":"","Column10":"","Column11":"sample cell,","Column12":"sample cell,","Column13":"","Column14":"sample cell,","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"","Column19":"","Column20":"sample cell,","Column21":"","Column22":"","Column23":"","Column24":"sample cell,","Column25":"sample cell,","Column26":"","Column27":"","Column28":"","Column29":"sample cell,","Column30":""},
{"Column1":"","Column2":"sample cell,","Column3":"sample cell,","Column4":"","Column5":"","Column6":"sample cell,","Column7":"","Column8":"sample cell,","Column9":"","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"","Column14":"sample cell,","Column15":"","Column16":"sample cell,","Column17":"","Column18":"","Column19":"sample cell,","Column20":"","Column21":"","Column22":"","Column23":"sample cell,","Column24":"sample cell,","Column25":"","Column26":"sample cell,","Column27":"sample cell,","Column28":"","Column29":"","Column30":""},
{"Column1":"","Column2":"sample cell,","Column3":"sample cell,","Column4":"sample cell,","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"sample cell,","Column10":"","Column11":"","Column12":"sample cell,","Column13":"sample cell,","Column14":"sample cell,","Column15":"","Column16":"sample cell,","Column17":"","Column18":"","Column19":"","Column20":"","Column21":"","Column22":"sample cell,","Column23":"","Column24":"sample cell,","Column25":"","Column26":"sample cell,","Column27":"","Column28":"sample cell,","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"","Column3":"sample cell,","Column4":"","Column5":"sample cell,","Column6":"","Column7":"sample cell,","Column8":"","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"sample cell,","Column17":"","Column18":"","Column19":"","Column20":"sample cell,","Column21":"","Column22":"","Column23":"sample cell,","Column24":"","Column25":"sample cell,","Column26":"sample cell,","Column27":"","Column28":"sample cell,","Column29":"","Column30":"sample cell,"},
{"Column1":"sample cell,","Column2":"sample cell,","Column3":"","Column4":"sample cell,","Column5":"sample cell,","Column6":"sample cell,","Column7":"","Column8":"","Column9":"","Column10":"","Column11":"sample cell,","Column12":"","Column13":"sample cell,","Column14":"sample cell,","Column15":"","Column16":"sample cell,","Column17":"sample cell,","Column18":"","Column19":"","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"","Column25":"sample cell,","Column26":"sample cell,","Column27":"sample cell,","Column28":"","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"sample cell,","Column3":"","Column4":"","Column5":"","Column6":"sample cell,","Column7":"","Column8":"","Column9":"sample cell,","Column10":"","Column11":"","Column12":"","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"sample cell,","Column17":"sample cell,","Column18":"","Column19":"","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"sample cell,","Column24":"","Column25":"","Column26":"","Column27":"sample cell,","Column28":"","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"","Column3":"sample cell,","Column4":"","Column5":"","Column6":"","Column7":"sample cell,","Column8":"","Column9":"sample cell,","Column10":"","Column11":"","Column12":"","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"sample cell,","Column19":"","Column20":"sample cell,","Column21":"sample cell,","Column22":"sample cell,","Column23":"","Column24":"","Column25":"","Column26":"","Column27":"sample cell,","Column28":"","Column29":"","Column30":""},
{"Column1":"","Column2":"sample cell,","Column3":"","Column4":"sample cell,","Column5":"sample cell,","Column6":"sample cell,","Column7":"sample cell,","Column8":"","Column9":"","Column10":"sample cell,","Column11":"sample cell,","Column12":"","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"","Column19":"","Column20":"","Column21":"sample cell,","Column22":"","Column23":"sample cell,","Column24":"sample cell,","Column25":"","Column26":"","Column27":"","Column28":"sample cell,","Column29":"sample cell,","Column30":"sample cell,"},
{"Column1":"","Column2":"","Column3":"sample cell,","Column4":"","Column5":"sample cell,","Column6":"","Column7":"sample cell,","Column8":"","Column9":"sample cell,","Column10":"","Column11":"sample cell,","Column12":"","Column13":"sample cell,","Column14":"","Column15":"","Column16":"","Column17":"sample cell,","Column18":"","Column19":"sample cell,","Column20":"","Column21":"","Column22":"sample cell,","Column23":"sample cell,","Column24":"sample cell,","Column25":"sample cell,","Column26":"sample cell,","Column27":"","Column28":"sample cell,","Column29":"","Column30":"sample cell,"},
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":"sample cell,","Column6":"","Column7":"sample cell,","Column8":"sample cell,","Column9":"sample cell,","Column10":"sample cell,","Column11":"sample cell,","Column12":"sample cell,","Column13":"sample cell,","Column14":"","Column15":"sample cell,","Column16":"","Column17":"sample cell,","Column18":"","Column19":"sample cell,","Column20":"sample cell,","Column21":"sample cell,","Column22":"","Column23":"","Column24":"sample cell,","Column25":"","Column26":"","Column27":"","Column28":"","Column29":"","Column30":""}
]
